ehitchhiking-ui/src/app/shared/services/authorization.service.ts

```diff
@@ -0,0 +1,66 @@
+import {UserService} from './user.service';
+import {URL_REGISTRY} from '@shared/constants/urlRegistry';
+import {NotificationService} from './notification.service';
+import {ApiService} from '@shared/services/api.service';
+import {Injectable} from '@angular/core';
+import {catchError, share} from 'rxjs/operators';
+import {HttpResponse, HttpErrorResponse} from '@angular/common/http';
+import {of} from 'rxjs';
+import {Router} from '@angular/router';
+
+@Injectable()
+export class AuthorizationService {
+	constructor(
+		private notificationService: NotificationService,
+		private apiService: ApiService,
+		private userService: UserService,
+		private router: Router
+	) {}
+
+	doAuthorization(login: string, password: string): void {
+		this.apiService
```

Comment:
pls add AuthorizationApiService that will be work with ApiService

Reply:
added